src/NzbDrone.Core/IndexerSearch/Definitions/AlbumSearchCriteria.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace NzbDrone.Core.IndexerSearch.Definitions
+{
+    public class AlbumSearchCriteria : SearchCriteriaBase
+    {
+        public override string ToString()
+        {
+            return $"[{Album.Title}]";
```

Comment:
Should we also include artist name when returning string? quite a few albums have same name.